compute/cker/include/cker/operation/optimized/BinaryArithmeticOps.h

```diff
@@ -213,22 +353,46 @@ inline void BroadcastAddFivefold(const BinaryArithmeticOpParam &params,
   }
 }
 
+inline void BroadcastAddDispatchQuant8(const BinaryArithmeticOpParam &params,
+                                       const Shape &input1_shape, const uint8_t *input1_data,
+                                       const Shape &input2_shape, const uint8_t *input2_data,
+                                       const Shape &output_shape, uint8_t *output_data)
+{
+  if (params.broadcast_category == BroadcastableOpCategory::kGenericBroadcast)
+  {
+    const std::function<uint8_t(const BinaryArithmeticOpParam &, const uint8_t &, const uint8_t &)>
+        fn =
+            [](const BinaryArithmeticOpParam &params, const uint8_t &a, const uint8_t &b) -> float {
```

Comment:
could you modify float -> uint8_t ?

Reply:
thanks for spotting :), made the change.